src/FsAutoComplete.Core/Commands.fs

```diff
@@ -358,17 +365,18 @@ type Commands<'analyzer> (serialize : Serializer, backgroundServiceEnabled) =
         return CoreResponse.ErrorRes ex.Message
     }
 
-    member private x.AsCancellable (filename : SourceFilePath) (action : Async<CoreResponse<'b>>) =
+    member private x.AsCancellable (filename : SourceFilePath) (action : Async<'t>) =
```

Comment:
I made this a bit more generic because I needed cancellation both for CoreResponse-returning members as well as just Asyncs in general. So CoreResponse-returning callsites got patched up to add a call to `recoverCancellation` as well to unwrap the cancellation error if necessary.